var/spack/repos/builtin/packages/dramsim2/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dramsim2(MakefilePackage):
+    """
+    DRAMsim is a hardware-validated, cycle-accurate
+    C based simulator for DRAM devices such as DDR3
+    """
+
+    git = "https://github.com/umd-memsys/DRAMSim2"
+
+    version('2.2',
+            url="https://github.com/dramninjasUMD/DRAMSim2/archive/v2.2.2.tar.gz",
```

Comment:
The URL says 2.2.2, but the version says 2.2. What's up with that? Also, can you move the URL up above like the git?